### PR TITLE
added AQA Lightning Talk Series link and Reproducing Test Failures Lo…

### PR DESCRIPTION
**Pull request by JayGreck**
…cally link in testJobTemplate
Fixes: #3401 
Signed-off-by: Jay <jay.greck01@btinternet.com>

### CODE REVIEW

**Comment by llxia** [comment]
ECA validation failed. @JayGreck Could you double-check your email and make sure it is the same as the one for ECA? Thanks

**Comment by JayGreck** [comment]
I did make a pull request earlier with the wrong email for the ECA by mistake, so i closed it and opened this new one. The email I entered for this pull request is the one I used for the ECA so I am unsure to why the ECA is failing.

**Comment by smlambert** [comment]
The ECA-signoff doc linked to from https://github.com/adoptium/aqa-tests/blob/master/Contributing.md#eclipse-contributor-agreement is a helpful reference when hitting ECA validation check failures.

**Comment by JayGreck** [comment]
@llxia I think the ECA has passed now. Thanks @smlambert for the helpful link to the guide for the ECA!

**Comment by llxia** [comment]
https://ci.adoptopenjdk.net/job/WIP_Test_Job_Auto_Gen/71/

**Comment by llxia** [comment]
https://ci.adoptopenjdk.net/job/Test_openjdk17_j9_sanity.perf_x86-64_linux/

image

**Review by llxia — APPROVED** [review]
Thanks @JayGreck

**Review by smlambert — CHANGES_REQUESTED** [review]
I've suggested a change, to remove the h3 tags around the word wiki, as that looks odd in the final generated job.

File: buildenv/jenkins/testJobTemplate [hunk 1]
```diff
@@ -183,7 +183,7 @@ ARCH_OS_LIST.each { ARCH_OS ->
...
+ 					description('<h1>THIS IS AN AUTOMATICALLY GENERATED JOB. PLEASE DO NOT MODIFY, IT WILL BE OVERWRITTEN.</h1><br><p>This job is defined in <a href="https://github.com/adoptium/aqa-tests/blob/master/buildenv/jenkins/testJobTemplate" target="_blank">testJobTemplate</a> in the https://github.com/adoptium/aqa-tests repo. If you wish to change the job, please modify testJobTemplate script. </p><p>For more information about job parameters, please refer <a href="https://github.com/adoptium/aqa-tests/wiki/How-to-Run-a-Grinder-Build-on-Jenkins" target="_blank">How to Run a Grinder Build on Jenkins</a></p><h3>wiki:</h3><p><a href="https://github.com/eclipse-openj9/openj9/wiki/AQA-Lightning-Talk-Series" target="_blank">AQA Lightning Talk Series</a></p><p><a href="https://github.com/eclipse-openj9/openj9/wiki/Reproducing-Test-Failures-Locally" target="_blank">Reproducing Test Failures Locally</a></p>')
```
[the remark this answers is not in the document]
Comment (smlambert) [reply buildenv/jenkins/testJobTemplate:186]: @JayGreck - can you make this final update to remove the `<h3>` tag around the word wiki please?  Once that is done, we can merge it in.

[the remark this answers is not in the document]
Comment (JayGreck) [reply buildenv/jenkins/testJobTemplate:186]: That's not a problem, I will get to work on that now

[the remark this answers is not in the document]
Comment (JayGreck) [reply buildenv/jenkins/testJobTemplate]: I've removed the <h3> tags and pushed the changes.

**Review by smlambert — APPROVED** [review]
thanks @JayGreck

**Pull request merged**